index.html

```diff
@@ -0,0 +1,235 @@
+    </div>
+
+
+
+    <!--  Section Two -->
+
+
+
+    <section class="Section-two-section">
+        <div class="section-two">
+            <div class="header">
+                <h1>How can we help you?</h1>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Suspendisse et<br> justo. Praesent mattis commodo augue.</p>
+            </div>
+
+
+            <div class="container">
+
+                <div class="box">
+
+                    <h2><i class="fas fa-object-ungroup"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-code"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-envelope-open-text"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+            </div>
+
+
+            <div class="container">
+
+                <div class="box">
+
+                    <h2><i class="fas fa-lightbulb"></i> <san></san>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-cash-register"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-user-tie"></i> spanDesign</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+            </div>
+
+        </div>
+    </section>
+
+
+    <section class="section-three-section">
+
+        <div class="section-three">
+            <div class="section-three-head">
+                <h1>Portfolio</h1>
+                <p>Senior graphic designer responsible for daily graphic design content for<br> a medium-sized firm. Worked regularly on graphic, layout, and<br> production materials.</p>
+            </div>
+        </div>
+
+        <div class="container-images">
+            <div><img src="images/pexels-photo-270348.jpeg" alt="">
```

Comment:
It is best practice to add image alt attributes. kindly give the alt attribute a suitable value.